recipes/CommonVoice/common_voice_prepare.py

```diff
@@ -304,6 +326,32 @@ def create_csv(
     logger.info(msg)
 
 
+def convert_mp3_to_wav(audio_mp3_path, sampling_rate=16_000):
+    """Convert an mp3 file to a wav file.
+
+    Parameters
+    ----------
+    audio_mp3_path : str
+        The path to the opus file to be converted.
```

Comment:
make sure to add sampling_rate in the docstring. Furthermore, maybe we should have a global variable at the top e.g. `VERBOSITY=0` which will by default remove ffmpeg output (so what you did with > /dev/null), otherwise, it will output in the stdout stream